cartographer/pose_graph/constraint/constraint_utils.h

```diff
@@ -25,6 +25,21 @@
 namespace cartographer {
 namespace pose_graph {
 
+#define FIND_NODE(node_name, node_id, map, log_message)                   \
```

Comment:
Maybe rename to FIND_NODE_OR_RETURN

Reply:
You could also pull the type out, so that it's clearer form the outside what is happening:
FIND_NODE_OR_RETURN(auto my_node, ...);